homeassistant/components/xiaomi_miio/select.py

```diff
@@ -34,23 +35,19 @@
 
 
 @dataclass
-class SelectorType:
-    """Class that holds device specific info for a xiaomi aqara or humidifier selectors."""
+class XiaomiMiioSelectDescription(SelectEntityDescription):
+    """A class that describes select entities."""
 
-    name: str = None
-    icon: str = None
-    short_name: str = None
-    options: list = None
-    service: str = None
+    options: tuple = ()
 
 
 SELECTOR_TYPES = {
-    FEATURE_SET_LED_BRIGHTNESS: SelectorType(
-        name="Led brightness",
+    FEATURE_SET_LED_BRIGHTNESS: XiaomiMiioSelectDescription(
+        key=ATTR_LED_BRIGHTNESS,
+        name="Led Brightness",
         icon="mdi:brightness-6",
-        short_name=ATTR_LED_BRIGHTNESS,
-        options=["Bright", "Dim", "Off"],
-        service=SERVICE_SET_LED_BRIGHTNESS,
+        device_class="xiaomi_miio__led_brightness",
+        options=("bright", "dim", "off"),
```

Comment:
This became a breaking change, as the options changed.

Reply:
Oh, you're right. `select` platform for `xiaomi_miio` is a new feature, users will start using it after 2021.8 release. Maybe we can change modes to lowercase in `rc` branch?

Reply:
I changed these names because of hassfest error:
```
[TRANSLATIONS] Invalid strings.select.json: Needs to be lowercase for dictionary value @ data['state']['xiaomi_miio__led_brightness']. Got {'Bright': 'Bright', 'Dim': 'Dim', 'Off': 'Off'}
```

Reply:
@bieniu Would this PR merge cleanly on rc branch? We could tag it for the release to avoid a breaking change later.

Reply:
@MartinHjelmare Yes, we can merge this PR on `rc` branch, I tested this change with beta 9 and it works as it should.

Reply:
Tagged it